classes/Commands/CheckRequirementsCommand.php

```diff
@@ -141,12 +141,49 @@ private function processRequirementErrors(): void
     private function processRequirementWarnings(): void
     {
         foreach ($this->upgradeSelfCheck->getWarnings() as $warning => $value) {
-            $wording = $this->upgradeSelfCheck->getRequirementWording($warning);
-            $this->writeWarning($wording['message']);
-            if (isset($wording['list'])) {
-                foreach ($wording['list'] as $item) {
-                    $this->output->writeln('    ' . $item);
-                }
+            switch ($warning) {
+                case UpgradeSelfCheck::CORE_TEMPERED_FILES_LIST_NOT_EMPTY:
+                    $wording = $this->upgradeSelfCheck->getRequirementWording($warning);
+                    $this->writeWarning($wording['message']);
+                    $missingFiles = $this->upgradeSelfCheck->getCoreMissingFiles();
+
+                    $this->output->writeln('    ' . count($missingFiles) . ' files are missing:');
+                    foreach ($missingFiles as $missingFile) {
+                        $this->output->writeln('        ' . $missingFile);
+                    }
```

Comment:
Command line take the advantage of tabulation in this case, can you try using them?

```suggestion
                    $this->output->writeln("\t" . count($missingFiles) . ' files are missing:');
                    foreach ($missingFiles as $missingFile) {
                        $this->output->writeln("\t\t" . $missingFile);
                    }
```